app/models/application.rb

```diff
@@ -13,7 +13,7 @@ class Application < ApplicationRecord
 
   has_paper_trail only: %i[lead_provider_approval_status participant_outcome_state]
 
-  belongs_to :user
+  belongs_to :user, touch: true
```

Comment:
could we limit this to when an application is accepted only?

Reply:
Updated the touch model to only update when `application.lead_provider_approval_status` changes.